src/evergreen/cli/main.py

```diff
@@ -371,6 +371,25 @@ def build_stats(ctx, build_id, tasks):
         click.echo(fmt_output(fmt, build.get_metrics().as_dict(include_children=tasks)))
 
 
+@cli.command()
+@click.pass_context
+@click.option("--project", required=True, help="The project name")
+@click.option("--commit", required=True, help="The full 40-char commit hash")
+def manifest(ctx, project, commit):
+    """Get a manifest for the given project and commit.
```

Comment:
Could you start the summary on a new line? `pydocstyle` doesn't like this format.

```
def manifest(ctx, project, commit):
    """
    Get a manifest for the given project and commit.
    ...
```